src/clientmap.cpp

```diff
@@ -293,7 +293,7 @@ void ClientMap::updateDrawList(video::IVideoDriver* driver)
 			float step = BS * 1;
 			float stepfac = 1.1;
 			float startoff = BS * 1;
-			float endoff = -BS*MAP_BLOCKSIZE * 1.42 * 1.42;
+			float endoff = -BS * MAP_BLOCKSIZE * 1.732050807569; // sqrt(3)
```

Comment:
sqrt(3) is no less "magic" to me than 1.732050807569 :D Is there another (short) way to explain what this number does?